### PR TITLE
Add mac id

### PR DESCRIPTION
**Pull request by nadouamanda**
add mac id to the commented area.

### CODE REVIEW

File: app/components/Feed.js [hunk 1]
```diff
@@ -13,6 +35,12 @@ export default class Feed extends Component {
     const timestamp = getDateFromTimestamp(this.props.post.timestamp);
 
     const avatarUrl = `https://avatars.dicebear.com/v2/avataaars/${userId}.svg`;
+    const currentUserAvatar = `https://avatars.dicebear.com/v2/avataaars/${this.props.currentUser}.svg`;
+
+    var jsx = [];
+    for (var i=0; i<this.state.comments.length; i++) {
+      jsx.push(<div className={`${style.comment}`}><div className={style.smallavatar}><img src={currentUserAvatar} alt="" /></div><div className={`${style.msg} ${style.text}`}>{this.state.comments[i]}</div></div>);
+    }
```
Comment (nadouamanda) [inline app/components/Feed.js:44]: allow user to post comments

File: app/components/Feed.js [hunk 2]
```diff
@@ -13,6 +35,12 @@ export default class Feed extends Component {
     const timestamp = getDateFromTimestamp(this.props.post.timestamp);
 
     const avatarUrl = `https://avatars.dicebear.com/v2/avataaars/${userId}.svg`;
+    const currentUserAvatar = `https://avatars.dicebear.com/v2/avataaars/${this.props.currentUser}.svg`;
+
+    var jsx = [];
+    for (var i=0; i<this.state.comments.length; i++) {
+      jsx.push(<div className={`${style.comment}`}><div className={style.smallavatar}><img src={currentUserAvatar} alt="" /></div><div className={`${style.msg} ${style.text}`}>{this.state.comments[i]}</div></div>);
```
Comment (philip-zhan) [inline app/components/Feed.js]: image

File: app/components/Board.js [hunk 3]
```diff
@@ -57,11 +57,21 @@ export default class Board extends Component {
   }
 
   render() {
+    var userId = "philzhan";
+
+    if ( require('getmac').isMac("ac:de:48:00:11:22") ) {
```
Comment (philip-zhan) [inline app/components/Board.js]: isMac() doesn't work becuase it returns true for any valid mac address, not just for your your mac address. 
image
image

**Review by philip-zhan — CHANGES_REQUESTED** [review]

File: app/components/Board.js [hunk 4]
```diff
@@ -57,11 +57,21 @@ export default class Board extends Component {
...
+      userId = "amandajiang";
+    } 
+
```
Comment (zhoucong96) [inline app/components/Board.js]: require('getmac').getMac(function(err, macAddress){
    if (err)  throw err
    console.log(macAddress)
})

**Review by philip-zhan — APPROVED** [review]

**Pull request merged**